src/Schema/Events/Click.ts

```diff
@@ -2061,3 +2061,95 @@ export interface ClickedUpdateArtwork {
   partner_id: string
   artwork_id: string
 }
+
+/**
+ * A user clicks on the read more link on the private artwork page.
+ *
+ * This schema describes events sent to Segment from [[ClickedOnReadMore]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOnReadMore",
+ *    context_module: "About the work" | "About the artist" | "Sidebar",
+ *    subject: "Read more"
+ *    type: "Link"
+ *  }
+ * ```
+ */
+export interface ClickedOnReadMore {
+  action: ActionType.clickedOnReadMore
+  context_module: string
+  subject: string
+  type: string
+}
+
+/**
+ * A user clicks on the learn more link on the private artwork page.
+ *
+ * This schema describes events sent to Segment from [[ClickedOnLearnMore]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOnLearnMore",
+ *    context_module: "Sidebar",
+ *    subject: "Learn more"
+ *    type: "Link"
+ *    flow: "Shipping" | "Artsy Guarantee"
+ *  }
+ * ```
+ */
+export interface ClickedOnLearnMore {
+  action: ActionType.clickedOnLearnMore
+  context_module: string
+  subject: string
+  type: string
+  flow: string
+}
+
+/**
+ * A user clicks on the Gallery name on the private artwork page.
+ *
+ * This schema describes events sent to Segment from [[ClickedOnGalleryName]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOnGalleryName",
+ *    context_module: "Sidebar",
+ *    subject: "Gallery Name"
+ *    type: "Link"
+ *    flow: "Exclusive Access"
+ *  }
+ * ```
+ */
+export interface ClickedOnGalleryName {
```

Comment:
We can just use a generic click event for this one! It sounds a bit too specific to track click gallery name

Reply:
Ok great! Removed this in the [newest PR](https://github.com/artsy/cohesion/pull/490). I also wanted to ask about whether we want to track clicking on the artist name like we do with the gallery name? That would probably also use the generic click event as well then, right?

The image below highlights what we are going to track with this 591 ticket, with the exception of the artist.

![Screenshot 2024-04-19 at 13 54 46](https://github.com/artsy/cohesion/assets/6280410/36f71f79-8f73-4056-95d8-65e38ff4964b)

Reply:
The artist name is not specific to the private artwork, so don't need to track for now!